sos/report/plugins/cloud_init.py

```diff
@@ -18,10 +18,15 @@ class CloudInit(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
     plugin_name = 'cloud_init'
     packages = ('cloud-init',)
     services = ('cloud-init',)
+    commands = ('cloud-init',)
 
     def setup(self):
+        self.add_cmd_output([
+            'cloud-init --version',
+            'journalctl --boot=0 -o short-precise'
```

Comment:
I'm not sure a full `journal` collection makes sense here - shouldn't this be limited to the `cloud-init` unit?